Sources/App/Resources/sessionConfig.json

```diff
@@ -29,7 +29,8 @@
                         "answer": {
                             "anyOf": [
                                 { "type": "string" },
-                                { "type": "number" }
+                                { "type": "number" },
+                                { "type": "null" }
                             ],
                             "description": "The patient's answer."
```

Comment:
It would make sense to explain what type should be used in which occasion, would be helpful for the model to understand this better right in the context.

Reply:
Added a small explanation

Reply:
Thank you!